sdjournal/journal_test.go

```diff
@@ -204,10 +204,7 @@ func TestJournalGetEntry(t *testing.T) {
 		t.Fatalf("Error writing to journal: %s", err)
 	}
 
-	r := j.Wait(time.Duration(1) * time.Second)
-	if r < 0 {
-		t.Fatalf("Error waiting to journal")
-	}
+	time.Sleep(time.Duration(1) * time.Second)
```

Comment:
We could loop over `sd_journal_wait()`/`sd_journal_get_data()` until we eventually get our message matching the field, but I agree this could end up in an endless loop.

Not really happy about the `time.Sleep` at all, so let's observe this in subsequent CI builds.